### PR TITLE
chore: update to 11

### PR DESCRIPTION
**Pull request by henryiii**
Attempted to add version 11 and bump 10 to 10.0.1. Please check. :)

### CODE REVIEW

File: 11.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+# Signing keys
+ENV GPG_KEYS 09C4E7007CB2EFFB A2C794A986419D8A B4468DF4E95C63DC D23DD2C20DD88BA2 8F0871F202119294 0FC3042E345AD05D
+
+# Retrieve keys
+RUN gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys $GPG_KEYS
+
+# Version info
+ENV LLVM_RELEASE 10
+ENV LLVM_VERSION 10.0.0
```
Comment (silkeh) [inline 11.Dockerfile]: This should be `11` and `11.0.0` respectively.

**Review by silkeh — CHANGES_REQUESTED** [review]
Thanks for this!

I think the `PLATFORM` for this release (in `install.sh`) should be:


```
PLATFORM="linux-gnu-ubuntu-20.04"
```

File: 11.Dockerfile (hunk 1 above)
Comment (henryiii) [reply 11.Dockerfile]: That's embarrassing, I was sure I'd changed this...

**Comment by silkeh** [comment]
LGTM, thanks!

**Pull request merged**